tests/sentry/api/serializers/test_project.py

```diff
@@ -748,6 +748,15 @@ def test_replay_rage_click_flag(self):
         result = serialize(self.project, self.user, DetailedProjectSerializer())
         assert result["options"]["sentry:replay_rage_click_issues"] is False
 
+    def test_replay_hydration_error_flag(self):
+        result = serialize(self.project, self.user, DetailedProjectSerializer())
+        # default should be true
+        assert result["options"]["sentry:replay_hydration_error_issues"] is True
+
+        self.project.update_option("sentry:replay_hydration_error_issues", False)
+        result = serialize(self.project, self.user, DetailedProjectSerializer())
+        assert result["options"]["sentry:replay_hydration_error_issues"] is False
```

Comment:
This is good and should be kept.  But we also need to call the API endpoint that returns these options so we know its being returned properly.